multinet/api.py

```diff
@@ -326,3 +328,37 @@ def delete_table(workspace: str, table: str) -> Any:
     """Delete a table."""
     db.delete_table(workspace, table)
     return table
+
+
+@bp.route("/uploads", methods=["POST"])
+def create_upload() -> str:
+    """Create a collection for multipart upload."""
+
+    sysdb = db.db("_system")
+
+    # note: arangodb collections can't start with a number
+    upload_id = f"upload_{uuid4().hex}"
+
+    sysdb.create_collection(upload_id)
```

Comment:
I think rather than put these collections at the top level of `sysdb`, there should be a collection there called `uploads`, and these collections should become documents within that collection.

Reply:
Done. Also resolves https://github.com/multinet-app/multinet-server/pull/448#discussion_r469379410